lib/src/main/java/org/team100/lib/config/FeedforwardConstants.java

```diff
@@ -6,9 +6,9 @@ public class FeedforwardConstants {
     private final double m_kSS;
     private final double m_kDS;
 
-    //Default no friction or attached objects contraints
+    //Default no friction or attached objects contraints for NEO
     public FeedforwardConstants() {
```

Comment:
don't make this a constructor, make a static method that says what it means, e.g. "public static FeedforwardConstants makeNeo() {return new FeedfowardConstants(0.122,0,0.1,0.065);}"

Reply:
I don't know why this still says it's not outdated but I fixed this